src/core/datastructures/camera/skewedperspectivecamera.cpp

```diff
@@ -113,11 +160,11 @@ mat4 SkewedPerspectiveCamera::calculateProjectionMatrix() const {
 
 void SkewedPerspectiveCamera::serialize(Serializer& s) const {
     Camera::serialize(s);
-    s.serialize("fovy", fovy_);
+    s.serialize("fov", fovy_);
```

Comment:
No. We have two field of views, one vertical and one horizontal (derived from the aspect ratio if I recall correctly). This is the vertical field of view. Could be better named

Reply:
only one is ever stored. the other one is never used, and if it would be, it would be computed with the vertical fov and aspect ratio. Renaming this makes it much easier to convert old workspaces since the vertical fov property has always been serialised as "fov" the camera it self has never been serialised before, so changing this here will not break anything.

Reply:
ok